
  taskspace_to_PSM1_tr: [0.06975183158467946, -0.044896462965694536, -0.1886528031817931, -0.0064811985949385165, -0.0018147878400363148, -0.00797855449467206, 0.9999455201549954]
  taskspace_to_PSM2_tr: [ 0.0983936, -0.069261, -0.119392, 0.00135533, -0.00201776, -0.0258034, 0.999664 ]

  taskspace_to_right_cam_transform: [0.0629172251323, -0.0327951270345, 0.524393155643, -0.0247808374108, 0.939524141022, -0.341087287698, -0.0184325989698]

  taskspace_to_left_cam_transform: [0.0501210519338 ,-0.0301954788821, 0.52824255036, -0.0104638987159, 0.93878630722, -0.343523002797, 0.0237217779568]

  left_cam_to_right_cam_transform: [-0.0303521, -0.00234023, -0.00311738, -0.00308049, 0.0455042, -0.000920183,   0.998959]


